docker-compose.yml

```
@@ -11,11 +11,14 @@ services:
       - docker-compose.env
 
   localstack:
-    image: localstack/localstack-full:0.12.8
+    image: localstack/localstack-full:1.4.0
```

`localstack-full` isn't a thing any more, just `localstack`:
```suggestion
    image: localstack/localstack:1.4.0
```